src/main/java/trial/Trial.java

```diff
@@ -0,0 +1,62 @@
+package trial;
+
+import java.util.Objects;
+
+public class Trial {
+
+    private final TrialOrder trialOrder;
+    private final int alivePins;
+
+    private Trial(final TrialOrder trialOrder, final int alivePins){
+        this.trialOrder = trialOrder;
+        this.alivePins = alivePins;
+    }
+
+    static Trial create(final TrialOrder trialOrder, final int alivePins){
+        return new Trial(trialOrder, alivePins);
+    }
+
+    static Trial createByFirst(final int alivePins){
+        return new Trial(TrialOrder.FIRST, alivePins);
+    }
+
+    static Trial createBySecond(final int alivePins){
+        return new Trial(TrialOrder.SECOND, alivePins);
+    }
+
+    static Trial createByThird(final int alivePins) {
+        return new Trial(TrialOrder.THIRD, alivePins);
+    }
+
+    public TrialResultType getResultType(){
+        if(trialOrder == TrialOrder.FIRST){
```

Comment:
```suggestion
        if(TrialOrder.isEqualTo(trialOrder)){
```
이런식으로 TrialOrder가 직접 판단하는 식이 어떨까요?